packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -142,9 +149,11 @@ const AccountWizard = observer(props => {
             }
             return previous_state;
         });
+        if (!state_items?.length) setRealAccountSignupFormData(getItems(get_items_props));
+
         setPreviousData(fetchFromStorage());
         setMounted(true);
-    }, [residence_list, states_list, fetchResidenceList, fetchStatesList, has_residence]);
+    }, [residence_list, states_list, fetchResidenceList, fetchStatesList, has_residence, setRealAccountSignupFormData]);
```

Comment:
Is it working as expected as I'm noticing the dependency array is missing `state_items`?

Reply:
@shahzaib-deriv yes, it works fine. This dependency array has many skipped values which there are inside of the `useEffect`. As I understood this `useEffect` needs just to put some data in state before mount of the component. But I'll try to add this `state_items` to array too and recheck how it works now

Reply:
@shahzaib-deriv I've add state_items and it works good, but linter wants to add these too : `setIsTradingAssessmentForNewUserEnabled, getData, get_items_props`, but when I did it this form doesn't work :) this component was created like this to skip some dependencies. So I've just added vars to dependency array which I've added to useEffect

Reply:
I've deleted `state_items` from the array, because tests failed